tests/phpunit/Checker/CLI_Runner_Tests.php

```diff
@@ -10,9 +10,16 @@
 use WordPress\Plugin_Check\Test_Data\Empty_Check;
 use WordPress\Plugin_Check\Test_Data\Error_Check;
 use WordPress\Plugin_Check\Test_Data\Runtime_Check;
+use WordPress\Plugin_Check\Test_Utils\Traits\With_Mock_Filesystem;
 
 class CLI_Runner_Tests extends WP_UnitTestCase {
 
+	use With_Mock_Filesystem;
+
+	public function set_up() {
+		$this->set_up_mock_filesystem();
+	}
```

Comment:
Same as above.

Reply:
See my comment on the above - https://github.com/10up/plugin-check/pull/173#discussion_r1206553708